yarn.lock

```diff
@@ -1282,6 +1282,16 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
+"@emotion/babel-preset-css-prop@^10.0.5":
```

Comment:
im not sure why our babel plugins were added to the lock 🤔

Reply:
It usually happens if some dependency is referencing a version different from latest, or when yarn shits its pants (it happens often)

Reply:
actually I've found why this has happened right now with a help of `yarn why` - those are dependencies of `gatsby-plugin-emotion`, so it makes sense that they have been added